src/medCore/gui/toolboxes/medToolBox.h

```diff
@@ -112,6 +117,21 @@ public slots:
     //! Switch between errors
     void handleDisplayError(int);
 
+    //! Deactivate toolbox
+    void setToolBoxOnWaitStatus();
+
+    //! Deactivate toolbox in function non using medRunnableProcess
+    void setToolBoxOnWaitStatusForNonRunnableProcess();
```

Comment:
This option shouldn't even be available because no toolbox should run a process in the foreground (registration blocks everything for example). I'll post an issue to remind us to fix all non-background running toolboxes.